<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>New Account</string>
    </property>
   </widget>
   <widget class="QFrame" name="WhatAccount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="GoButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="maxVisibleItems">
      <number>5</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>GoButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
